clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/StreamInvocationMessage.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+package com.microsoft.aspnet.signalr;
+
+public class StreamInvocationMessage extends HubMessage {
+
+    int type = HubMessageType.STREAM_INVOCATION.value;
+    String invocationId;
```

Comment:
Remember, unlike in C#, the default access for Java is "package" which is more like `internal` than `private`. These are truly `private` so we need to explicitly make them `private`